tools/encodings/GF 2016 Glyph Sets/FilterLists/Google Latin Pro.txt

```diff
@@ -372,6 +372,15 @@ breveinvertedcomb
 dblgravecomb
 horncomb
 dotbelowcomb
+strokeshortcomb
+brevecomb_acutecomb
+brevecomb_gravecomb
+brevecomb_hookabovecomb
+brevecomb_tildecomb
+circumflexcomb_acutecomb
+circumflexcomb_gravecomb
+circumflexcomb_hookabovecomb
+circumflexcomb_tildecomb
```

Comment:
I found 'circumflexcomb_acutecomb' *very* useful for design-cases where custom Vietnamese accents are requiered. All these characters were borrowed from the Vietnamese set in Glyph App.

Technically they aren't necessary for font generation.
However, I think specifying them here will encourage designers to pay more attention
to Vietnamese.  Or at least list them in the recommended section, and hope that people _will_ read it.

@crystaltype What do you think?

<img width="845" alt="screen shot 2016-07-20 at 12 21 50" src="https://cloud.githubusercontent.com/assets/16634089/16994284/b61a13ee-4e74-11e6-8c1e-edf74e611ed2.png">